themes/_core/js/product.js

```diff
@@ -220,6 +220,7 @@ function updateProduct(event, eventType, updateUrl) {
         $(prestashop.selectors.product.customization)
           .first()
           .replaceWith(data.product_customization);
+        $('.product-actions input[name=id_customization]').val(0);
```

Comment:
```suggestion
        $(`$(prestashop.selectors.product.actions) input[name=id_customization]`).val(0);
```

Reply:
@NeOMakinG Should we add a new selector ?

Reply:
Of course we should add a new selector to the selector's map!

Reply:
Thanks @NeOMakinG 

So @Hlavtox could you add a new selector for this input (prestashop.selectors.product.input_customization for example) ?